splunk_topology/conf.yaml.example

```diff
@@ -42,6 +42,7 @@ instances:
     # snapshot: true
     # polling_interval_seconds: 300
     # saved_searches_parallel: 5
+    ignore_saved_search_errors: true
```

Comment:
what happens when I do not specify this value, omit this setting because it is a default, will it continue to work as expected?

Reply:
by default, I have set this config as true. 

`self.splunk_ignore_config = instance.get('ignore_saved_search_errors', 'true')`